meta/Makefile

```diff
@@ -76,6 +76,7 @@ OBJ = saimetadata.o saimetadatautils.o saiserialize.o
 SYMBOLS = $(OBJ:=.symbols)
 
 all: saisanitycheck saimetadatatest saiserializetest saidepgraph.svg $(SYMBOLS)
+	./checkheaders.pl ../inc ../inc
```

Comment:
same folder?

Reply:
Yes since Sai dont know any other folder and this check checks od script compiles